gcloud/credentials.py

```diff
@@ -207,6 +217,9 @@ def _get_signature_bytes(credentials, string_to_sign):
         pkey = _get_pem_key(credentials)
         if not isinstance(string_to_sign, six.binary_type):
             string_to_sign = string_to_sign.encode('utf-8')
+        if crypto is None:
+            raise EnvironmentError('pyOpenSSL must be installed to sign '
```

Comment:
OK, will do. I'm going to squash and fold this change into the final (squashed commit) and then will merge once tests pass. LMK if this is no bueno.